code/src/components/GamePage.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { Direction } from "./Direction";
+import { EndPage } from "./EndPage";
+import map from "images/map.png";
+import Lottie from "react-lottie";
+import animationData from "../lotties/loading";
+import animationDataShip from "../lotties/ship";
+import animationDataWater from "../lotties/water";
+
+export const GamePage = () => {
+  const labyrinth = useSelector((store) => store.labyrinth.location);
+  const loading = useSelector((store) => store.ui.loading);
+  const [setDirection] = useState("");
+
```

Comment:
You missed to declare direction `const [direction, setDirection] = useState("");`